book/src/pickers.md

```diff
@@ -0,0 +1,11 @@
+## Using pickers
+
+Helix has a variety of pickers, which are interactive windows used to select various kinds of items. These include a file picker, global search picker, and more. Most pickers are accessed via keybindings in [space mode](./keymap.md#space-mode). Pickers have their own [keymap](./keymap.md#picker) for navigation.
+
+### Filtering Picker Results
+
+Most pickers perform fuzzy matching using [fzf syntax](https://github.com/junegunn/fzf?tab=readme-ov-file#search-syntax). Two exceptions are the global search picker, which uses regex, and the workspace symbol picker, which passes search terms to the LSP.
```

Comment:
There's an exception to the fzf syntax - the OR syntax is not implemented yet per discussion on #11245

Reply:
Is there an issue specifically for OR support? I couldn't find it.

Reply:
Not that I'm aware of either outside of #11245.